modules/aws-kubernetes/versions.tf

```diff
@@ -0,0 +1,13 @@
+terraform {
+  required_version = ">= 0.13"
```

Comment:
should this be 0.15 instead?

Reply:
Technically there should not be any syntactical differences between 0.13.x and 0.15.x so what should work with 0.13.x should work as well with 0.15.x. I can update it to be more strict about version requirement though.

Reply:
If the migration from 0.13 to 0.15 is painless I'd say let's put the higher one as requirement, so when we have to upgrade to newer versions we are closer, i.e. it will be easier to migrate from 0.15 to 1 than from 0.13 to 1.